nexus/Freqlog/Freqlog.py

```diff
@@ -346,8 +357,21 @@ def start_logging(self, new_word_threshold: float | None = None, chord_char_thre
                       f"allowed_chars={self.allowed_chars}, "
                       f"allowed_first_chars={self.allowed_first_chars}, "
                       f"modifier_keys={self.modifier_keys}")
-        self.listener.start()
-        self.mouse_listener.start()
+
+        def log_start():
+            if self.loggable:
```

Comment:
I'm pretty sure this isn't required since `loggable` will always be `True` in a `start_logging` call. Double-check that though, before removing this if it's indeed true.

Reply:
It's not guaranteed if the class is initialized with it False.

Reply:
I'm pretty sure that in that case, `start_logging` will never be called, no? Since this code is in `start_logging`, where `loggable` should be `True`

Reply:
Before start_logging is called, it can be set to False, thus leading to a bug. I think it's better to be safe than sorry and leave this here.

Reply:
Okay I'm fine with adding this safeguard, let's have the whole `self._log_start` function return immediately if `loggable` is `False` though. (See associated pull method out comment)